### DISCUSSION TITLE
Upgrade via Git to a specific version

### DISCUSSION THREAD

**Discussion by muttandy** [snipe/snipe-it#11850] [Q&A] [answered]
Hi

I'm just wondering, if it's possible to upgrade Snipe-IT via GIT on Windows Server to a speciific version, not to the latest ?

Thanks a lot in advance

Andy

**Comment by aquadiode** [comment, marked as answer]
I'm not 100% sure about this but the git part of php upgrade.php is just doing a git checkout and takes a parameter.  You can upgrade on the development branch with `php upgrade.php develop` so I think you can just get the tag you want with `php upgrade.php tags/v6.0.10` or the commit sha you want like `php upgrade.php 687a764`.  In git, this will put you in a headless state which I think is generally not a problem if you aren't making changes to the files.

**Reply by muttandy** [reply]
Great, I have to try that next time, thanks a lot !
